bin/dart_sass_embedded.dart

```diff
@@ -55,13 +55,8 @@ void main(List<String> args) {
           _decodeImporter(dispatcher, request, importer) ??
           (throw mandatoryError("Importer.importer")));
 
-      var globalFunctions = request.globalFunctions.map((signature) {
-        try {
-          return hostCallable(dispatcher, functions, request.id, signature);
-        } on sass.SassException catch (error) {
-          throw paramsError('CompileRequest.global_functions: $error');
-        }
-      });
+      var globalFunctions = request.globalFunctions.map((signature) =>
+          hostCallable(dispatcher, functions, request.id, signature));
```

Comment:
This should probably wrap the error with a message like "Error parsing global function:"

Reply:
I think this is fine as below is what the dart-sass throw:

```
> const sass = require('sass')
undefined
> sass.compileString('a {b: c}', { functions: { 'a': function () {} } })
Uncaught:
sass.Exception [Error]: Invalid signature "a": expected "(".
  ╷
1 │ a
  │  ^
  ╵
  - 1:2  root stylesheet
    at Object.wrapException (/home/opc/sass/node_modules/sass/sass.dart.js:1247:17)
    at Object.throwWithTrace0 (/home/opc/sass/node_modules/sass/sass.dart.js:23150:15)
    at ScssParser0.wrapSpanFormatException$1$1 (/home/opc/sass/node_modules/sass/sass.dart.js:90334:13)
    at ScssParser0.wrapSpanFormatException$1 (/home/opc/sass/node_modules/sass/sass.dart.js:90340:19)
    at ScssParser0.parseSignature$1$requireParens (/home/opc/sass/node_modules/sass/sass.dart.js:94260:19)
    at ScssParser0.parseSignature$0 (/home/opc/sass/node_modules/sass/sass.dart.js:94263:19)
    at _parseFunctions_closure0.call$2 (/home/opc/sass/node_modules/sass/sass.dart.js:79087:58)
    at Object.jsForEach (/home/opc/sass/node_modules/sass/sass.dart.js:22726:18)
    at Object._parseFunctions0 (/home/opc/sass/node_modules/sass/sass.dart.js:17593:9)
    at StaticClosure.compileString0 (/home/opc/sass/node_modules/sass/sass.dart.js:17491:17)
```

Reply:
Actually let me add `Invalid signature` to the error message